scilpy/viz/scene_utils.py

```diff
@@ -481,4 +486,406 @@ def render_scene(scene, window_size, interactor,
+
+    Parameters
+    ----------
+    img_count : int
+        Image count to be arranged in the mosaic.
+    rows : int
+        Row count.
+    cols : int
+        Column count.
+    """
+
+    cell_count = rows * cols
+
+    if img_count < cell_count:
+        raise ValueError(
+            f"Less slices than cells requested.\nImage count: {img_count}; "
+            f"Cell count: {cell_count} (rows: {rows}; cols: {cols}).\n"
+            "Please provide an appropriate value for the rows, cols for the "
+            "slice count.")
+    elif img_count > cell_count:
+        raise ValueError(
+            f"More slices than cells requested.\nImage count: {img_count}; "
+            f"Cell count: {cell_count} (rows: {rows}; cols: {cols}).\n"
+            "Please provide an appropriate value for the rows, cols for the "
+            "slice count.")
+
+
+def rgb2gray4pil(rgb_arr):
+    """Convert an RGB array to grayscale and convert to `uint8` so that it can
+    be appropriately handled by `PIL`.
+
+    Parameters
+    ----------
+    rgb_arr : ndarray
+        RGB value data.
+
+    Returns
+    -------
+    Grayscale `unit8` data.
+    """
+
+    def _rgb2gray(rgb):
+        img = Image.fromarray(np.uint8(rgb * 255)).convert("L")
+        return np.array(img)
+
+    # Convert from RGB to grayscale
+    gray_arr = _rgb2gray(rgb_arr)
+
+    # Relocate overflow values to the dynamic range
+    return (gray_arr * 255).astype("uint8")
+
+
+def create_image_from_scene(scene, size, mode=None, cmap_name=None):
+    """Create a `PIL.Image` from the scene data.
+
+    Parameters
+    ----------
+    scene : ndarray
+        Scene data.
+    size : array-like
+        Image size (pixels) (width, height).
+    mode : str, optional
+        Type and depth of a pixel in the `Pillow` image.
+    cmap_name : str, optional
+        Colormap name.
+
+    Returns
+    -------
+    image : PIL.Image
+        Image.
+    """
+
+    _arr = scene
+    if cmap_name:
+        # Apply the colormap
+        cmap = plt.get_cmap(cmap_name)
+        # data returned by cmap is normalized to the [0,1] range: scale to the
+        # [0, 255] range and convert to uint8 for Pillow
+        _arr = (cmap(_arr) * 255).astype("uint8")
+
+    # Need to flip the array due to some bug in the FURY image buffer. Might be
+    # solved in newer versions of the package.
+    image = Image.fromarray(_arr, mode=mode).transpose(Image.FLIP_TOP_BOTTOM)
+
+    return image.resize(size, Image.ANTIALIAS)
+
+
+def create_mask_from_scene(scene, size):
+    """Create a binary `PIL.Image` from the scene data.
+
+    Parameters
+    ----------
+    scene : ndarray
+        Scene data.
+    size : array-like
+        Image size (pixels) (width, height).
+
+    Returns
+    -------
+    image : PIL.Image
+        Image.
+    """
+
+    _bin_arr = scene > 0
+    _bin_arr = rgb2gray4pil(_bin_arr) * 255
+    image = create_image_from_scene(_bin_arr, size)
+
+    return image
+
+
+def draw_scene_at_pos(
```

Comment:
You can use the full line for the args of the def.

Reply:
Automatic code formatting breaks it this way. Avoids the user needing to manually break, the user breaking the line at times when the full length has not been used, etc.